### ISSUE TITLE
ability to resize reference image plus or minus the canvas size

### ISSUE THREAD

**Issue by essteewhy** [blocktronics/moebius#162]

**Comment by ollipelkonen** [comment]
If the exact opposite would help, I added some simple new features for reference images.
namely, "Fit to screen" and "Center position", and also "paste from a clipboard" as a bonus.
https://github.com/ollipelkonen/moebius/tree/reference-image_fixes

**Comment by bart-d** [comment]
> If the exact opposite would help, I added some simple new features for reference images.
> namely, "Fit to screen" and "Center position", and also "paste from a clipboard" as a bonus.

pasting the image is nifty :) not sure fit/center are of much help here but with what you've got you could easily add zoom in/out and change the position in such a way that it becomes very useful for those that use this feature.

**Comment by ollipelkonen** [comment]
Yes, scaling itself is easy to implement, but the hard part here's how to control it.
Some Photoshop-style stretching is quite out of a question now, but popup with scale inputs/sliders is not very intuitive to use

**Comment by bart-d** [comment]
> Yes, scaling itself is easy to implement, but the hard part here's how to control it.
> Some Photoshop-style stretching is quite out of a question now, but popup with scale inputs/sliders is not very intuitive to use

Just keyboard shortcuts to reduce or increase scale and move into 4 directions already would go a long way I guess. Would also prevent the user from stretching the image.

**Comment by ollipelkonen** [comment]
![Screenshot from 2020-10-26 13-22-17](https://user-images.githubusercontent.com/7767052/97166826-8ccd6d00-178e-11eb-822e-22e50d836d62.png)
committed
